test/browser/browser.sh

```diff
@@ -1,22 +1,15 @@
 #!/bin/sh
-set -eux
-
-export TEST_BROWSER=${TEST_BROWSER:-firefox}
 
-TESTS="$(realpath $(dirname "$0"))"
-export SOURCE="$(realpath $TESTS/../..)"
-export LOGS="$(pwd)/logs"
-mkdir -p "$LOGS"
-chmod a+w "$LOGS"
+set -eux
+cd "${0%/*}/../.."
 
-# install firefox (available everywhere in Fedora and RHEL)
-# we don't need the H.264 codec, and it is sometimes not available (rhbz#2005760)
-dnf install --disablerepo=fedora-cisco-openh264 -y --setopt=install_weak_deps=False firefox
+# HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2033020
+dnf update -y pam || true
```

Comment:
Why did you introduce that now? That bug got fixed half a year ago..

Reply:
copypasta from other projects...

Reply:
removed again